apps/oxlint/src/tester.rs

```diff
@@ -1,11 +1,13 @@
 #[cfg(test)]
-use crate::cli::{LintRunner, lint_command};
+use std::{env, path::PathBuf};
+
 #[cfg(test)]
 use cow_utils::CowUtils;
 #[cfg(test)]
 use lazy_regex::Regex;
+
 #[cfg(test)]
-use std::{env, path::PathBuf};
+use crate::cli::{LintRunner, lint_command};
```

Comment:
There should be a line break after this line. See "Line breaks" section in #12723.

> The last `use` / `mod` statement should have a line break after it, before any other code (including comments or `#[...]` attributes).